README.md

```diff
@@ -3,6 +3,9 @@
 ### What is this repo for
 This repo is created for saving resources we found useful by to date. Mostly contains **React** and **Web development** resources.
 
```

Comment:
Need header

Reply:
What kind of header? and What text should it contain?